docs/security/rbac.md

```diff
@@ -366,6 +366,42 @@ You can create additional policies to further limit user access. Use the web UI
     ]
 }
 ```
+## Multiple Resources Statements
+
+Starting from version v1.54.0, (for on-prem users using fluffy > v0.11.0) LakeFS supports specifying multiple resources in a single RBAC statement.
+To enable this, the resource field should contain a string representing a JSON-encoded list of resources.
+
+```json
+{
+    "statement": [
+        {
+            "action": [
+                "fs:Read*"
+            ],
+            "effect": "allow",
+            "resource": "[\"arn:lakefs:fs:::repository/repo1\",\"arn:lakefs:fs:::repository/repo2\"]"
+        }
+    ]
+}
+```
+Note: Ensure that the list is properly JSON-encoded as a string—using double quotes and escaping as shown—otherwise, the policy will not be parsed correctly.
```

Comment:
A tech worker once told me "you use the word note too much". I think that's a problem here, too.
```suggestion
The list must be properly encoded as a JSON string: quote each resource, and escape those quotes as shown. Otherwise, the policy cannot be parsed.
```